### DISCUSSION TITLE
Request: navigation_sc_request Command Parameters

### DISCUSSION THREAD

**Discussion by yakeer** [timdorr/tesla-api#530] [Q&A]
Anyone knows the Request Parameters for the "navigation_sc_request" command endpoint?

I believe that SC stands for SuperCharger.

**Comment by dtpanic** [comment]
I have the same question.

I want to navigate to a specific supercharger. I can call this endpoint and it returns 'success' no matter what parameters I pass. The car navigation then just sites at 'calculating'......
